scripts/change_url

```diff
@@ -109,13 +33,6 @@ ynh_script_progression --message="Starting a systemd service..." --weight=1
 
 ynh_systemd_action --service_name=$app --action="start" --log_path="/var/log/$app/$app.log"
```

Comment:
It doesn't seem relevant / necessary to keep this service start (and the stop before) if only the nginx conf is modified ?